Makefile

```diff
@@ -0,0 +1,29 @@
+DOCKER_COMPOSE_FILENAME=docker-compose.yml
+PHP_FPM_SERVICE_NAME=php
+
+.PHONY: shell
+shell:
+	docker compose -f ${DOCKER_COMPOSE_FILENAME} exec ${PHP_FPM_SERVICE_NAME} bash
```

Comment:
I will get rid of this `-f ${DOCKER_COMPOSE_FILENAME}` part. It is not needed because there are not other docker compose files.
Also we can remove this variable.

Reply:
removed